main/src/cgeo/geocaching/gcvote/GCVote.java

```diff
@@ -42,6 +43,54 @@ private GCVote() {
         // utility class
     }
 
+    private static class SingletonHolder {
+        private final static GCVote INSTANCE = new GCVote();
```

Comment:
There is no need to create a singleton here if the only thing that is done with it is calling a static method which could be called on the class itself.

Please remove this singleton, and call `GCVote.login()` directly.